src/assets/i18n/de.json

```diff
@@ -150,6 +150,19 @@
 		"editMemberInstitute": "Bearbeite die Mitglieder der Institution.",
 		"membersHaveAccess": "Diese Mitglieder haben Zugriff auf die Institution.",
 		"addMember": "Mitglied hinzufügen",
+		"addMember_text": "Neue Mitarbeiter müssen ein OEB-Konto anlegen und verifizieren, bevor sie zu einem Emittenten hinzugefügt werden.",
+		"addMember_chooseRole": "Rolle wählen",
+		"addMember_owner": "Owner",
+		"addMember_ownerText": "Möglichkeit, Mitarbeiter hinzuzufügen und zu entfernen. Volle Rechte zum Erstellen, Löschen und Verleihen von Abzeichen. Möglichkeit, Ausstellerdetails zu bearbeiten.",
```

Comment:
If we use "Owner" in german and english, the translation is unnecessary. Also some of the translations are not used at the moment. Please either remove them or add a TODO comment, where the translations should be used in the future

Reply:
Thanks Jonas! 
I translated owner, editor and staff member. For now only their texts are translated but not used as I wanted to keep the original texts for future. I also added a comment that mentioned texts are ready to be referenced from `i18n`